### ISSUE TITLE
Telegram join channel/group check doesn't work

### ISSUE THREAD

**Issue by coinmetaverse** [mujadmani/Easy-Telegram-Airdrop-Bot#5]
It is true . There is this problem

I don't know why but telegram join channel/group check doesn't work for me.
I think the problem is in [this](https://github.com/mujadmani/Easy-Telegram-Airdrop-Bot/blob/7538348e94947f5cef86fd5c6554e4dc06ac0170/bot.py#L280) function.
I also have an errors in heroku log

**Comment by lexciobotariu** [comment]
have you invited your bot to your group and channel?

**Comment by atta1177** [comment]
yes same probleam bro i make bot admin on channel & group but same error + please make note about how to install on vps
